xc/device_common.ml

```diff
@@ -38,6 +38,7 @@ exception Hotplug_script_expecting_field of device * string
 exception Unknown_device_type of string
 exception Unknown_device_protocol of string
 exception QMP_Error of int * string
+exception QMP_connection_error
```

Comment:
Adding the connection path or domid would make the exception more useful

Reply:
I add domid and error message for this. 
Updated.